packages/adapter-netlify/index.js

```diff
@@ -63,6 +63,12 @@ function validate_config() {
 			);
 		}
 
+		if (netlify_config.redirects && netlify_config.redirects.from === '/*') {
```

Comment:
I haven't tested this, but I think `netlify_config.redirects` is an array, so you might need a `forEach`?

Reply:
I think the second half of this check should be removed. see my comment above

Reply:
Makes sense. In which case just checking if it's defined is enough.

That said, would a better approach be to define the adapter's catch-all in `netlify.toml` instead of in `_redirects`? That way user-defined redirects in both locations would continue to work (so long as they were not catch-all).

Reply:
I played around with modifying the netlify.toml but then realized that it would never go into the final build directory (and modifying the original config file is out of question).
So either we instruct the users to set it up themselfes or we use this automatic approach but then disallowing redirects in the toml. For now I'd say supporting at least `_redirects` is better than nothing. What do you think?

Reply:
Definitely better than nothing, yes.